launcher.py

```diff
@@ -13,14 +18,15 @@ def clear_screen():
 clear_screen()
 
 # VERSION
-launcher_version = str("0.2-pre")
+launcher_version = str("0.1a")
```

Comment:
Please do not change the version number.

Reply:
i don't remember changing